packages/vite/src/node/server/middlewares/transform.ts

```diff
@@ -111,7 +111,7 @@ export function transformMiddleware(
       // since we generate source map references, handle those requests here
       if (isSourceMap) {
         const originalUrl = url.replace(/\.map($|\?)/, '$1')
-        const map = (await moduleGraph.getModuleByUrl(originalUrl))
+        const map = (await moduleGraph.getModuleByUrl(originalUrl, false))
```

Comment:
why is the parameter `false` here and below? this is a node middleware, so wouldn't it only be used in an SSR context? I'd expect it to be `true` instead. if `false` is the correct value it could potentially be worth extracting to a shared variable and adding a comment

Reply:
Unless I'm missing something this middleware is used in dev/serve mode, and not for SSR. E.g. below call to `transformRequest` will resolve/load/transform with ssr=false.

Reply:
yeah, I think you're right. thanks